### PR TITLE
order for unbind vars

### PR DESCRIPTION
**Pull request by mjudeikis**
@kargakis FIY

### CODE REVIEW

File: ci-operator/templates/openshift/openshift-azure/build-node-image.yaml [hunk 1]
```diff
@@ -131,9 +131,10 @@ objects:
 
         # Get resource group and image name
         source /tmp/shared/environment
-        echo "RESOURCEGROUP=${RESOURCEGROUP_PREFIX}-image-build"
-        echo "IMAGE_NAME=${IMAGE_NAME}"
         export AZURE_OREG_URL='registry.access.redhat.com/openshift3/ose-${component}:${version}'
```
Comment (thekad) [inline ci-operator/templates/openshift/openshift-azure/build-node-image.yaml]: I know this isn't part of your change, but I'm fairly sure this line is wrong (unless there's a shell setting somewhere I am not seeing), see:

```
[kad@kenpachi release (upstream)]$ component=foo
[kad@kenpachi release (upstream)]$ version=bar
[kad@kenpachi release (upstream)]$ echo $component 
foo
[kad@kenpachi release (upstream)]$ echo $version 
bar
[kad@kenpachi release (upstream)]$ export AZURE_OREG_URL='registry.access.redhat.com/openshift3/ose-${component}:${version}'
[kad@kenpachi release (upstream)]$ echo $AZURE_OREG_URL 
registry.access.redhat.com/openshift3/ose-${component}:${version}
```

tl;dr single quotes won't process variable interpolation
